src/ru/2018-01/08/02.md

```diff
@@ -1,7 +1,20 @@
 ---
-title:  Урок
+title:  Совместная поддержка миссии
 date:   18/02/2018
 ---
 
-### <center>Мы подготавливаем данный урок</center>
-<center>Попробуйте позже</center>
\ No newline at end of file
+Иисус повелел нам: «Проповедуйте Евангелие» (`Мк. 16:15`), а также: «Научите все народы… уча их соблюдать всё» (`Мф. 28:19, 20`). Таким образом, Бог желает, чтобы мы были вовлечены в самую важную работу на земле — привлечение людей к Иисусу. На управителе лежит ответственность материально поддерживать эту миссию за счет ресурсов, которые доверил ему Бог. Участие в этом деле усиливает чувство личного долга представлять Христа другим людям. Каждый ученик, управитель и работник должен приносить всю десятину целиком для этой священной работы. Нам следует молиться о единстве, чтобы мы были верны в финансировании миссии. И успех миссии будет укреплять наше единство в вере.
+
+**Какой финансовый план для осуществления этой миссии одобрил Бог?** 
```

Comment:
@KeyJoo In this case both Line 8 and 9 should be within the comment block, which is between ``` ` ``` symbols